sqlpage/templates/tracking.handlebars

```diff
@@ -0,0 +1,20 @@
+<div class="card">
+  <div class="card-body">
+	 <div class="d-flex align-items-center">
+		<div class="subheader">{{title}}</div>
+	 </div>
+	<div class="d-flex align-items-baseline">
+		<div class="h1 mb-3 me-2">{{#if information}}{{information}}{{/if}}</div>
+	 </div>
+	 <div class="d-flex align-items-baseline">
+		<div class="mb-3 me-2">{{#if description}}{{description}}{{/if}}{{#if description_md}}{{{markdown description_md}}}{{/if}}</div>
```

Comment:
You can use the [`remove-bottom-margin`](https://github.com/lovasoa/SQLpage/blob/main/sqlpage/sqlpage.css#L15) class here to prevent the double margin (from the div and from the `<p>`) when using markdown.